### ISSUE TITLE
libmc_user.so patching: also possible for wavpack?

### ISSUE THREAD

**Issue by Soukyuu** [diorcety/mazda3#13]
Do you think it would be possible to add wavpack support the same way you added flac support? That would be rather awesome.

**Comment by diorcety** [comment]
Yes it's possible. You need the gstream library for wavpack and add metadata parser for it. (Look for how it's done with flac)

**Comment by Soukyuu** [comment]
Do I have to compile it myself? For what architecture? ARMv7? Or are there pre-built libs? 
Maybe you could take a look at it when you have some free time?

**Comment by diorcety** [comment]
Yes you have to programm and build it yourself. Do a file command on object files in this repository you will have information about the arch. No sorry I have currently no time at all to do that
